### ISSUE TITLE
Doesn't work after browser restart

### ISSUE THREAD

**Issue by ttuygun** [acrogenesis/jsonview-safari#15]
If I re-enable/re-install the extension, the extension has works otherwise not working (browser restart g.e.).

**Comment by acrogenesis** [comment]
I believe this is because the extension isn't signed by Apple.

**Comment by ttuygun** [comment]
I don't think so. Signing or unsigning is factor when the add-on installing. After the installation there is no auto-stop to add-on by Apple.
